components/wysiwyg/widget-block/index.js

```diff
@@ -6,11 +6,11 @@ import { belongsToACollection } from 'components/collections-panel/collections-p
 
 import { connect } from 'react-redux';
 
-import * as actions from './widget-block-actions';
-import * as reducers from './widget-block-reducers';
-import initialState from './widget-block-default-state';
+import * as actions from './actions';
+import * as reducers from './reducers';
+import initialState from './default-state';
```

Comment:
Let's go for `initial-state`, please. https://github.com/resource-watch/resource-watch#modules